docs/tutorials/developer-journey/level-4/4.3-ckbtc-and-bitcoin.md

```diff
@@ -0,0 +1,351 @@
+
+:::info
+In this tutorial and other pages of the IC developer documentation, note that we use the lowercase version of **bitcoin** to refer to the token BTC, and use the uppercase version **Bitcoin** to refer to the Bitcoin network itself. This capitalization distinction is in accordance with the Bitcoin style guide.
+:::
+
+## Bitcoin integration architecture
+
+Through the IC's integration with the Bitcoin network, canisters deployed on the IC are able to receive, hold, and send bitcoin. Each of these functions utilize transactions on the Bitcoin network, and allow canisters to act as if they are users on the Bitcoin network. 
+
+At the core of this integration is a novel protocol for chain-key signatures that is based on **threshold ECDSA**. Threshold ECDSA is a protocol suite that implements chain-key signatures on the IC. In this suite, there are several features that include key generation, periodic key re-sharing, computing pre-signatures, XNet key re-sharing, and public key retrieval. 
+
+On a high level, subnets that have ECDSA enabled hold a threshold ECDSA master key, which is generated using the IC's key generation protocol. A master key can be used to generate canister ECDSA keys. It is important to know that an ECDSA private key only exists in a secret-shared form during it's lifetime, whether it is being generated, shared within a subnet, or shared from one subnet to another. 
+
+We'll dive deeper into threshold ECDSA in a future tutorial. You can also take a deep dive in the [technology overview to learn more](/docs/current/developer-docs/integrations/t-ecdsa/t-ecdsa-how-it-works). 
+
+On the protocol-level, the Bitcoin integration and chain-key ECDSA each expose an API on the IC management canister. Remember that the IC management canister isn't a canister but a facade that is used to enable functionality for management operations such as starting or stopping a canister. While the Bitcoin API is specific to the Bitcoin network, the chain-key ECDSA signature API can be used for other workflows, such as the integration with other blockchains like Ethereum.
+
+For a canister to make a Bitcoin transaction, a request must be sent to a Bitcoin-enabled subnet. On that subnet, a Bitcoin canister is implemented as a typical NNS-managed canister that is accessible through the management canister Bitcoin API. The Bitcoin canister holds the IC's on-chain Bitcoin state, which includes the UTXO set, the most recent Bitcoin blocks, and the current outgoing transactions being sent to the Bitcoin network. When a canister submits a transaction to the Bitcoin management canister API, the transaction is queued to be submitted to the Bitcoin network. To send transactions to the Bitcoin network, an adapter connects to the nodes of the Bitcoin network and functions similarly to a traditional Bitcoin node. In every subnet round, the Bitcoin adapter obtains the pending transactions and queues them for being submitted asynchronously. 
```

Comment:
It would be helpful to add a link to the wiki article for the interested reader: https://wiki.internetcomputer.org/wiki/Bitcoin_Integration

Reply:
Revised to include this resource